README.md

```diff
@@ -36,14 +36,13 @@ You can find a full running example here [Cloudflare Playwright running example]
 ```js
 import { launch } from '@cloudflare/playwright';
 
-const todos = searchParams.getAll('todo');
 
 const browser = await launch(env.MYBROWSER);
```

Comment:
We should wrap it in:

```ts
export default {
  async fetch(request, env): Promise<Response> {
    // ...
  }
} satisfies ExportedHandler<Env>;
```

Otherwise copy & paste of that code won't work.